textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/fuzzytokenmatcher/FuzzyTokenMatcherTest.java

```diff
@@ -0,0 +1,186 @@
+package edu.uci.ics.textdb.dataflow.fuzzytokenmatcher;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.queryparser.classic.ParseException;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.IField;
+import edu.uci.ics.textdb.api.common.IPredicate;
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.common.Schema;
+import edu.uci.ics.textdb.api.storage.IDataWriter;
+import edu.uci.ics.textdb.common.constants.DataConstants;
+import edu.uci.ics.textdb.common.constants.SchemaConstants;
+import edu.uci.ics.textdb.common.constants.TestConstants;
+import edu.uci.ics.textdb.common.exception.DataFlowException;
+import edu.uci.ics.textdb.common.field.DataTuple;
+import edu.uci.ics.textdb.common.field.DateField;
+import edu.uci.ics.textdb.common.field.DoubleField;
+import edu.uci.ics.textdb.common.field.IntegerField;
+import edu.uci.ics.textdb.common.field.ListField;
+import edu.uci.ics.textdb.common.field.Span;
+import edu.uci.ics.textdb.common.field.StringField;
+import edu.uci.ics.textdb.common.field.TextField;
+import edu.uci.ics.textdb.dataflow.common.FuzzyTokenPredicate;
+import edu.uci.ics.textdb.dataflow.utils.TestUtils;
+import edu.uci.ics.textdb.storage.DataStore;
+import edu.uci.ics.textdb.storage.writer.DataWriter;
+
+/**
+ * @author Parag Saraogi
+ *
+ */
+
+public class FuzzyTokenMatcherTest {
+
+    private FuzzyTokenMatcher fuzzyMatcher;
+    private IDataWriter dataWriter;
+    private DataStore dataStore;
+    private Analyzer analyzer;
+    
+    @Before
+    public void setUp() throws Exception {
+        dataStore = new DataStore(DataConstants.INDEX_DIR, TestConstants.SCHEMA_PEOPLE);
+        analyzer = new StandardAnalyzer();
+        dataWriter = new DataWriter(dataStore, analyzer);
+        dataWriter.clearData();
+        dataWriter.writeData(TestConstants.getSamplePeopleTuples());
+    }
+    
+    @After
+    public void cleanUp() throws Exception {
+        dataWriter.clearData();
+    }
+    
+    public List<ITuple> getResults(String query,double threshold, ArrayList<Attribute> attributeList, boolean getSpan) throws DataFlowException, ParseException {
```

Comment:
Please take a look at other test cases (e.g., DictionaryMatcherTest) to see the general structure of a test case.  Also I feel this test case needs to cover more cases. The current one is too simple.